en-US/emails/2018/firefox_learning_journey.lang

```diff
@@ -0,0 +1,308 @@
+
+
+;Happy customizing!
+Happy customizing!
+
+
+# Link: https://addons.mozilla.org/firefox/addon/emoji-cheatsheet/
+;Improving your emojis
+Improving your emojis
+
+
+# CTA
+;Get the Extensions
+Get the Extensions
+
+
+# Email: Advocacy fundraising
+# Mockup image: https://goo.gl/zhvqko
+# Subject
+;The big-picture story of Mozilla
+The big-picture story of Mozilla
+
+
+# Pre-header text
+;What kind of future internet do you want? Walled gardens and locked gates, or a great wide open for all to explore?
+What kind of future internet do you want? Walled gardens and locked gates, or a great wide open for all to explore?
+
+
+;Hello There,
+Hello There,
+
+
+;While you may think of Mozilla as just a pioneer of open-source software, there's so much more to our story.
+While you may think of Mozilla as just a pioneer of open-source software, there's so much more to our story.
+
+
+# Link: https://donate.mozilla.org/
+;<a>Mozilla's mission is to be the not-for-profit champion of a healthy, safe and open internet. We ask for your help to ensure those principles always ring true.</a>
+<a>Mozilla's mission is to be the not-for-profit champion of a healthy, safe and open internet. We ask for your help to ensure those principles always ring true.</a>
+
+
+;Mozilla believes we should all be free to explore and discover, create and innovate, without barriers or limitations. We are an independent voice helping to ensure that power is in the hands of the many, not held by the few.
+Mozilla believes we should all be free to explore and discover, create and innovate, without barriers or limitations. We are an independent voice helping to ensure that power is in the hands of the many, not held by the few.
+
+
+;We need your support to carry this healthy internet mission forward. We all want an internet where our safety, security and identity are respected — now and for generations to come. Your help makes a difference.
+We need your support to carry this healthy internet mission forward. We all want an internet where our safety, security and identity are respected — now and for generations to come. Your help makes a difference.
+
+
+;While Firefox development is supported entirely by our commercial partnerships, we rely upon donations from folks like you to support the Mozilla Foundation's initiatives like our advocacy work for net neutrality and copyright reform; our Tech Policy fellowships to steer policy towards a healthy internet; and our programs to teach web literacy around the world.
+While Firefox development is supported entirely by our commercial partnerships, we rely upon donations from folks like you to support the Mozilla Foundation's initiatives like our advocacy work for net neutrality and copyright reform; our Tech Policy fellowships to steer policy towards a healthy internet; and our programs to teach web literacy around the world.
+
+
+# Link: https://donate.mozilla.org/
+;<a>Please join us by making a donation today.</a>
```

Comment:
Check the error page… (translation is different from the original, and we don't need tags)